lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -206,68 +215,91 @@ <h3 class="course-title">
                         </a>
                     % endif
                 % endif
-
             % endif
 
-            <div class="wrapper-action-more" data-course-key="${enrollment.course_id}">
-              <button type="button" class="action action-more" id="actions-dropdown-link-${dashboard_index}" aria-haspopup="true" aria-expanded="false" aria-controls="actions-dropdown-${dashboard_index}" data-course-number="${course_overview.number}" data-course-name="${course_overview.display_name_with_default}" data-dashboard-index="${dashboard_index}">
-                <span class="sr">${_('Course options for')}</span>
-                <span class="sr">&nbsp;
-                  ${course_overview.display_name_with_default}
-                </span>
-                <span class="fa fa-cog" aria-hidden="true"></span>
-              </button>
-              <div class="actions-dropdown" id="actions-dropdown-${dashboard_index}" tabindex="-1">
-                <ul class="actions-dropdown-list" id="actions-dropdown-list-${dashboard_index}" aria-label="${_('Available Actions')}" role="menu">
-                  % if can_unenroll:
-                    <li class="actions-item" id="actions-item-unenroll-${dashboard_index}" role="menuitem">
-                      <% course_refund_url = reverse('course_run_refund_status', args=[unicode(course_overview.id)]) %>
+            % if not entitlement:
+                <div class="wrapper-action-more" data-course-key="${enrollment.course_id}">
+                  <button type="button" class="action action-more" id="actions-dropdown-link-${dashboard_index}" aria-haspopup="true" aria-expanded="false" aria-controls="actions-dropdown-${dashboard_index}" data-course-number="${course_overview.number}" data-course-name="${course_overview.display_name_with_default}" data-dashboard-index="${dashboard_index}">
+                    <span class="sr">${_('Course options for')}</span>
+                    <span class="sr">&nbsp;
+                      ${course_overview.display_name_with_default}
+                    </span>
```

Comment:
The screenreader text may be positioned incorrectly in some languages. Instead of translating the first half of the sentence and appending the display name to the end, translate the entire sentence using placeholders for the additional `span` tags and `display_name`, then interpolate the values into the string.

Reply:
These are all from previous code, I think I'm going to open a new ticket to fix those issues as opposed to trying to shove new bug fixes in here.